src/libsystemd/sd-bus/bus-control.c

```diff
@@ -459,9 +459,13 @@ _public_ int sd_bus_get_name_creds(
         if (!BUS_IS_OPEN(bus->state))
                 return -ENOTCONN;
 
-        /* Only query the owner if the caller wants to know it or if
-         * the caller just wants to check whether a name exists */
-        if ((mask & SD_BUS_CREDS_UNIQUE_NAME) || mask == 0) {
+        /* If the name is unique anyway, we can use it directly */
+        if (!unique && name[0] == ':')
```

Comment:
Bad rebase? `unique` is initalized to NULL above and not changed up until this point.